### ISSUE TITLE
MDF V4 methods use deprecated numpy tostring() method

### ISSUE THREAD

**Issue by rentauskasm** [danielhrisca/asammdf#524]
# Python version
Output:
```
xls is not supported
xlsx is not supported
yaml is not supported
('python=3.6.8 (tags/v3.6.8:3c6b436a57, Dec 23 2018, 23:31:17) [MSC v.1916 32 '
 'bit (Intel)]')
'os=Windows-10-10.0.17763-SP0'
'numpy=1.19.5'
'asammdf=6.2.0'
```

The latest version of asammdf uses the ```numpy``` method ```tostring()``` in asammdf.blocks.v4_blocks, which has been deprecated in favor of the ```to_bytes()``` method, see here: <https://numpy.org/doc/stable/reference/generated/numpy.ndarray.tostring.html>

Usually I wouldn't make a ticket for this, but when used to convert ```.asc``` files to ```.mf4``` files, it is printed tens of thousands of times depending on the size of the ```.asc``` file. This method may also be in uses in other parts of the module, but this is the main one where I have observed this behavior.

**Comment by danielhrisca** [comment]
Hello @rentauskasm 
I've replace ``tostring`` with ``tobytes`` in the development branch. It will be included in the `6.3.0` release

**Comment by rentauskasm** [comment]
Nice, thanks!
